hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/S3InMemoryCache.java

```diff
@@ -60,23 +71,8 @@ public S3SecretValue get(String id) {
    * Builder for {@link S3InMemoryCache}.
    */
   public static class S3InMemoryCacheBuilder {
```

Comment:
Thank you @ArafatKhan2198 for the patch. 

We can remove this builder: `S3InMemoryCache`.